### DISCUSSION TITLE
Welcome to teaproject Discussions!

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by waaghree** [comment]
Al here!  Wonderful to join up the community!

**Reply by kevingzhang** [reply]
The warmest welcome to you.

**Reply by kevingzhang** [reply]
Where are a lot of ongoing discussion. You can ask questions too.
